src/app/cli/src/services/accounts/models.rs

```diff
@@ -62,7 +62,9 @@ impl CurrentAccountIndication {
     }
 
     pub fn to_current_account_subject(&self) -> CurrentAccountSubject {
-        CurrentAccountSubject::logged(AccountName::from(self.account_name.clone()))
+        let is_admin = false;
+
+        CurrentAccountSubject::logged(AccountName::from(self.account_name.clone()), is_admin)
```

Comment:
If I understand correctly this path is used by CLI. Don't we want the user launching e.g. `kamu ui` to be considered an admin of their own local workspace?

Reply:
fair point, thought about it too, but decided that to be an admin you need explicit `is_admin` :thinking: 

yes, it would be better to give admin rights in this case.

Reply:
Fixed via 6b6b5932a59fc506238d95c3b369b715f664ba56

Reply:
It might be a good idea if a single-tenant repository user (`"kamu"`) is an admin.
But for a multi-tenant workspace I expected this marker to come from `UsersConfig`, so that we can configure both admin and non-admin users for our demos, and that would imply having a `false` value by default, unless specified in configuration explicitly.

Reply:
This was actually very explicitly written in the ticket:

> UsersConfig structure should hold optional isAdmin marker, false by default. It should define the built-in account’s data

Reply:
> But for a multi-tenant workspace I expected this marker to come from UsersConfig, so that we can configure both admin and non-admin users for our demos, and that would imply having a false value by default, unless specified in configuration explicitly
> ...
> This was actually very explicitly written in the ticket:

Yes, and it's actually done

```rust
#[derive(Debug, Clone, Eq, PartialEq, Serialize, Deserialize)]
#[serde(deny_unknown_fields, rename_all = "camelCase")]
pub struct AccountInfo {
    pub account_id: AccountID,
    pub account_name: AccountName,
    pub account_type: AccountType,
    pub display_name: AccountDisplayName,
    pub avatar_url: Option<String>,
    #[serde(default)]
    pub is_admin: bool, // <--- NEW
}
```
(c) https://github.com/kamu-data/kamu-cli/pull/414/files#diff-26be9abe208e5f3eed1d8c5b313725b924d46958959f284abc4738b71f98e060R30-R35

This way, we can set administrator rights for desired user(s):
```yaml
---
version: 1
kind: CLIConfig
content:
  engine:
    runtime: podman
  users:
    predefined:
      - accountId: 1337
        accountName: s373r
        displayName: Dima Pristupa
        accountType: User
        avatarUrl: https://avatars.githubusercontent.com/u/50896974?s=200&v=4
        isAdmin: true # <--- NEW: `false` if not present
```

Reply:
Done